### ISSUE TITLE
The use of dataset

### ISSUE THREAD

**Issue by zhangts1** [rainofmine/Bi-box_Regression#6]
I want to use a Caltech Pedestrian dataset for training, but for the transformation of the data set don't correspond with code format and don't know you is how to convert the data set.How to write in CSV, has met the needs of the code format?

**Comment by msha096** [comment]
I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.

Then I write code to transfer to the format I want to feed this model.

**Comment by zhangts1** [comment]
Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please send me the code of your conversion format for my reference? Thank you very much!

**Comment by zhangts1** [comment]
Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please send me the code of your conversion format for my reference? Thank you very much!

> I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> 
> Then I write code to transfer to the format I want to feed this model.

**Comment by zhangts1** [comment]
> 我使用加州理工学院的数据集进行培训。我的方法不方便，我使用matlab工具箱首先将.vbb和.seq文件传输到jpg和.txt / .csv。（有很多关于此的博客）。（https://blog.csdn.net/a2008301610258/article/details/45873867）我读过并引用的一个博客。
> 
> 然后我编写代码以转换为我想要提供此模型的格式。

Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please send me the code of your conversion format for my reference? Thank you very much!

**Comment by zhangts1** [comment]
> 我使用加州理工学院的数据集进行培训。我的方法不方便，我使用matlab工具箱首先将.vbb和.seq文件传输到jpg和.txt / .csv。（有很多关于此的博客）。（https://blog.csdn.net/a2008301610258/article/details/45873867）我读过并引用的一个博客。
> 
> 然后我编写代码以转换为我想要提供此模型的格式。

Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please share the code of your conversion format? Thank you very much!

**Comment by zhangts1** [comment]
> 我使用加州理工学院的数据集进行培训。我的方法不方便，我使用matlab工具箱首先将.vbb和.seq文件传输到jpg和.txt / .csv。（有很多关于此的博客）。（https://blog.csdn.net/a2008301610258/article/details/45873867）我读过并引用的一个博客。
> 
> 然后我编写代码以转换为我想要提供此模型的格式。



> I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> 
> Then I write code to transfer to the format I want to feed this model.

Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please share the code of your conversion format? Thank you very much!

**Comment by zhangts1** [comment]
> I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> 
> Then I write code to transfer to the format I want to feed this model.

Thank you very much for your reply. I have used the method mentioned above before.Now I have extracted.jpg and.txt with the code given by the official website of California institute of technology. Could you please share the code of your conversion format? Thank you very much!

**Comment by zhangts1** [comment]
> I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> 
> Then I write code to transfer to the format I want to feed this model.

I have seen the blog you quoted. Does this blog have only the annotations of the whole pedestrian box and no annotations of the visible part?

**Comment by msha096** [comment]
> > I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> > Then I write code to transfer to the format I want to feed this model.
> 
> I have seen the blog you quoted. Does this blog have only the annotations of the whole pedestrian box and no annotations of the visible part?

Yes the blog only have full box, you need to change a little bit in the official Matlab tool box to get visible box info.
Changing format code is really case by case and every time I will change the code to fit different  requirements.

**Comment by zhangts1** [comment]
> > > I used the Caltech dataset for training. My method is not convenient, I use the matlab toolbox to transfer .vbb and .seq file to jpg and .txt/.csv first. (There are lots of blogs about this). (https://blog.csdn.net/a2008301610258/article/details/45873867) one of the blogs I read and referenced.
> > > Then I write code to transfer to the format I want to feed this model.
> > 
> > 
> > I have seen the blog you quoted. Does this blog have only the annotations of the whole pedestrian box and no annotations of the visible part?
> 
> Yes the blog only have full box, you need to change a little bit in the official Matlab tool box to get visible box info.
> Changing format code is really case by case and every time I will change the code to fit different requirements.

Thank you very much for your help!
